Fluid.MvcViewEngine/FluidRendering.cs

```diff
@@ -71,17 +74,18 @@ static FluidRendering()
                 context.AmbientValues.Add("Body", body);
                 var layoutTemplate = ParseLiquidFile((string)layoutPath, fileProvider, false);
 
-                return await layoutTemplate.RenderAsync(context); 
+                return await layoutTemplate.RenderAsync(context);
             }
 
             return body;
         }
 
+
```

Comment:
Remove the extra line

Reply:
I did this as part of a follow up commit. Thanks for taking a look.